### PR TITLE
Updating Tests For Namespace Recommendations 

### PR DESCRIPTION
**Pull request by shekhar316**
## Description

This PR updates the tests for namespace recommendations with updated format. 

### Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Docs update
- [ ] Breaking change (What changes might users need to make in their application due to this PR?)
- [ ] Requires DB changes

**Test Configuration**
* Kubernetes clusters tested on: kruize-lm

## Checklist :dart:

- [x] Followed coding guidelines
- [x] Comments added
- [ ] Dependent changes merged
- [ ] Documentation updated
- [x] Tests added or updated

## Additional information

quay.io/rh-ee-shesaxen/autotune:ns-extype-change

### CODE REVIEW

File: tests/scripts/local_monitoring_tests/rest_apis/test_create_experiment.py [hunk 1]
```diff
@@ -113,17 +113,17 @@ def test_create_exp_valid_tests(test_name, expected_status_code, version, experi
...
-        ("invalid_container_exp_type_with_only_namespace", ERROR_500_STATUS_CODE, "v2.0", "tfb-workload-namespace", "default", "resource-optimization-local-monitoring", "monitor", "local", "prometheus-1", "container", None, None, None, "default", None, None, "15min", "0.1")
+        ("invalid_namespace_exp_without_exp_type", ERROR_STATUS_CODE, CREATE_EXP_CONTAINER_EXP_CONTAINS_NAMESPACE, "v2.0", "tfb-workload-namespace", "default", "resource-optimization-local-monitoring", "monitor", "local", "prometheus-1", None, None, None, None, "default", None, None, "15min", "0.1"),
+        ("invalid_both_container_and_namespace_without_exp_type", ERROR_STATUS_CODE, CREATE_EXP_CONTAINER_EXP_CONTAINS_NAMESPACE,"v2.0", "tfb-workload-namespace", "default", "resource-optimization-local-monitoring", "monitor", "local", "prometheus-1", None, "deployment", "tfb-qrh-sample", "default", "default", "kruize/tfb-qrh:1.13.2.F_et17", "tfb-server", "15min", "0.1"),
```
Comment (chandrams) [inline tests/scripts/local_monitoring_tests/rest_apis/test_create_experiment.py:119]: @shekhar316 Can you submit the tests with your build on jenkins and also add a test to test namespace exp with remote exp

Comment (shekhar316) [reply tests/scripts/local_monitoring_tests/rest_apis/test_create_experiment.py:119]: Results for Openshift - https://ci.app-svc-perf.corp.redhat.com/job/ExternalTeams/job/Autotune/job/kruize_functional_tests/153/

Results for Minikube - https://ci.app-svc-perf.corp.redhat.com/job/ExternalTeams/job/Autotune/job/kruize_functional_tests/155/

Comment (shekhar316) [reply tests/scripts/local_monitoring_tests/rest_apis/test_create_experiment.py:119]: Added a test for remote monitoring

Comment (chandrams) [reply tests/scripts/local_monitoring_tests/rest_apis/test_create_experiment.py:119]: I still see a failure in https://ci.app-svc-perf.corp.redhat.com/job/ExternalTeams/job/Autotune/job/kruize_functional_tests/155/, can you check

**Review by chandrams — APPROVED** [review]
LGTM. We can go ahead and merge these changes while we continue to investigate the minikube issue with load not being sufficient for cpu recommendations to be generated.

**Comment by dinogun** [comment]
@shekhar316 Please fix the conflicts

**Comment by shekhar316** [comment]
> @shekhar316 Please fix the conflicts

Rebased the PR.

**Pull request merged**